build.gradle

```diff
@@ -14,6 +14,8 @@ repositories {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-rx2:1.3.0'
+    implementation "io.reactivex.rxjava2:rxjava:2.2.7"
```

Comment:
Unfortunately there is not an API for interacting with RxJava 3 at this time. Once `kotlinx-coroutines-rx2` is updated to work with RxJava 3, this dependency should be removed.